extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts

```diff
@@ -276,6 +281,22 @@ export class GmailElementReplacer extends WebmailElementReplacer {
     return !!$('iframe.pgp_block').filter(':visible').length;
   };
 
+  private replaceActionsMenu = () => {
+    const gmailActionsMenuContainer = $(this.sel.msgActionsMenu);
+    if ($('.action_menu_message_button').length <= 0) {
+      const menuSecureReplyBtn = $(this.factory.actionsMenuBtn('reply')).insertAfter($(gmailActionsMenuContainer).find('#r')); // xss-safe-factory
+      const menuSecureForwardBtn = $(this.factory.actionsMenuBtn('forward')).insertAfter($(gmailActionsMenuContainer).find('#r3')); // xss-safe-factory
+      menuSecureReplyBtn.on(
+        'click',
+        Ui.event.handle((el, ev: JQuery.Event) => this.actionActivateSecureReplyHandler(el, ev))
+      );
+      menuSecureForwardBtn.on(
+        'click',
+        Ui.event.handle((el, ev: JQuery.Event) => this.actionActivateSecureReplyHandler(el, ev))
+      );
+    }
```

Comment:
adding reply and forward buttons are similar, let's make a separate helper method to remove code duplication:

```ts
const addMenuButton = (action: 'reply' | 'forward', selector: string) => {
    const target = gmailActionsMenuContainer.find(selector);
    const button = $(this.factory.actionsMenuBtn(action)).insertAfter(target); // xss-safe-factory
    button.on('click', Ui.event.handle((el, ev: JQuery.Event) => this.actionActivateSecureReplyHandler(el, ev)));
  };
```

Reply:
Thanks Roma for reviewing this. I'll follow the changes you've requested shortly.

Reply:
Done, thanks!